Re-add sizes.raw to the photo JSON

skyschub opened the pull request:
... because we need it for showing the raw image in the lightbox. This got lost, as the photo extension was made after API development started.

There are no integration tests for the lightbox, and since I don't want to spend two days on hotfixing a regression, we won't have integration tests after this PR either. But at least we have spec coverage now, so.. yay?

SuperTux88 reviewed (approved):
Nice catch :cookie:

diaspora-code-review commented on app/presenters/photo_presenter.rb:
```diff
@@ -12,7 +12,8 @@ def base_hash
         medium: url(:thumb_medium),
-        large:  url(:scaled_full)
+        large:  url(:scaled_full),
+        raw:    url,
```
Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.

denschub replied:
That's what I get for fixing the stylebot, eh?

diaspora-code-review commented on app/presenters/photo_presenter.rb:
```diff
@@ -26,7 +27,8 @@ def as_api_json(full=false)
         medium: url(:thumb_medium),
-        large:  url(:scaled_full)
+        large:  url(:scaled_full),
+        raw:    url,
```
Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.

skyschub committed:
Re-add sizes.raw to the photo JSON

because we need it for showing the raw image in the lightbox. This got lost, as the photo extension was made after API development started.

skyschub committed:
Do not allow additional properties in the API's photo_sizes schema definition
